.github/workflows/pr.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       with:
         workflow: test.yml
         repo: hashicorp/consul-k8s-workflows
-        ref: main
+        ref: update-bats-tests
```

Comment:
until the PR against the workflows repo merges this PR will be in draft but is still fine for review

Reply:
updated because the workflow PR has merged